content/en/blog/contributor-contest/index.md

```diff
@@ -0,0 +1,133 @@
+date = 2019-04-20
+lastmod = 2019-04-20
+
+draft = false
+
+# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
+authors = ["KubeEdge"]
+
+# Tags and categories
+# For example, use `tags = []` for no tags, or the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
+tags = ["KubeEdge", "kubeedge", "edge computing", "kubernetes edge computing", "K8S edge orchestration", "edge computing platform", "KubeCon", "cloud native"]
+categories = ["General", "Announcements"]
+summary = "To encourage open source community members participation in KubeEdge project, this contest is been conducted. KubeEdge maintainers welcome community members, individuals, organizations & students for this contest. The event will start on 20th April 2019 and end on 23rd May 2019"
+
+# Projects (optional).
+#   Associate this post with one or more of your projects.
+#   Simply enter your project's folder or file name without extension.
+#   E.g. `projects = ["deep-learning"]` references 
+#   `content/project/deep-learning/index.md`.
+#   Otherwise, set `projects = []`.
+# projects = ["internal-project"]
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder. 
+[image]
+  # Caption (optional)
+  caption = ""
+
+  # Focal point (optional)
+  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
+  #focal_point = "Center"
++++
+
+  
+
+## **How to participate ?**
+
+{{% alert note %}}
+*Category 1*
+
+1. Raise pull request (PR) either for feature development / test code development (may be unit test code, edge module test code or end to end test code) in repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+2. Identify defects, raise issues in respective repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+3. Re-solving existing issues in repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+4. Share requirements by creating issues in repo [kubeedge](https://github.com/kubeedge/kubeedge).
+{{% /alert %}}
+
+{{% alert note %}}
+*Category 2*
+
+Writing blogs about KubeEdge either in the [website](https://github.com/kubeedge/website) (on PR approved & merged, this gets published in kubeedge.io website blog) or in other popular technical blogging site.
```

Comment:
It's not clear how popular the technical blogging site should be. Maybe just say other technical blogging site.

Reply:
Done